modules/serverless-sinkbinding-ossm.adoc

```diff
@@ -0,0 +1,168 @@
+// Module included in the following assemblies:
+//
+// * /serverless/eventing/event-sources/serverless-custom-event-sources.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-sinkbinding-ossm_{context}"]
+= Integrating {SMProductShortName} with SinkBinding
+
+.Prerequisites
+
+* You have integrated {SMProductShortName} with {ServerlessProductName}.
+
+.Procedure
+
+. Create a `Service` in a namespace that is member of the `ServiceMeshMemberRoll`.
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: <namespace> <1>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <2>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> Injects {SMProductShortName} sidecars into the Knative service pods.
+
+. Apply the `Service` resource.
++
+[source,terminal]
+----
+$ oc apply -f <filename>
+----
+
+. Create a `SinkBinding`.
++
+[source,yaml]
+----
+apiVersion: sources.knative.dev/v1
+kind: SinkBinding
+metadata:
+  name: bind-heartbeat
+  namespace: <namespace> <1>
+spec:
+  subject:
+    apiVersion: batch/v1
+    kind: Job <2>
+    selector:
+      matchLabels:
+        app: heartbeat-cron
+
+  sink:
+    ref:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+      name: event-display
+----
+<1> A namespace that is member of the `ServiceMeshMemberRoll`.
+<2> In this example, any Job with the label `app: heartbeat-cron` will be bound to the event sink.
```

Comment:
Just present tense: "... heartbeat-cron` is bound..."

Reply:
Fixed!